proxy/prefect_flows.py

```
@@ -334,39 +374,51 @@ def deployment_schedule_flow_v4(
-            sleep(30)  # wait for 30 seconds after airbyte task
+def _run_tasks_with_sync_tolerance(tasks: list):
+    """Run all airbyte syncs first (collecting errors), then run the rest sequentially"""
+    sync_tasks = [t for t in tasks if _is_airbyte_sync_task(t)]
+    other_tasks = [t for t in tasks if not _is_airbyte_sync_task(t)]
+
+    # run all airbyte syncs, collecting errors
+    sync_errors = []
+    for task_config in sync_tasks:
+        try:
+            _run_task(task_config)
+        except Exception as error:  # skipcq PYL-W0703
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)